docs/readmes/orc8r/how_to_deploy_using_juju.md

```diff
@@ -0,0 +1,103 @@
+---
+id: how_to_deploy_using_juju
+title: How-To: Deploy Orchestrator using Juju (Beta)
+hide_title: true
+---
+
+# How-To: Deploy Orchestrator using Juju (Beta)
```

Comment:
the `(Beta)` is pretty unspecific. Can you explain in more details whether the deployment is `experimental` or `production ready` and whether it is `feature complete`? Maybe also link to the repository where the development of the juju deployment is happening.

Reply:
I added reference to the project's homepage / repository and also mentioned that it is not yet production ready. I don't think I should add more info as to **why** in the how-to guide as the project evolves every week and a how-to guide shouldn't contain such information. But for reference, here are the main reasons:
- It does not include integration with logging and monitoring tools (prometheus, alertmanager, grafana, fluentd, elasticsearch)
- It does not support scaling of components.

Reply:
> [...] as the project evolves every week [...]

but it contain a "what to expect right now (at time of writing)". If there are limitations and missing features, they should be listed in the beginning so that one can understand whether this deployment matches the expectations or is suitable for the use-case.

Since it is documentation about another project (https://github.com/canonical/charmed-magma) it might be an alternative to have just a "what to expect and where to find documentation" that points to somewhere else.

Reply:
I want to stress the fact that this new document is a how-to guide, not a reference document. I don't want to bundle different types of documentations into one place just because they relate to a same subject. If we need reference documentation on what is this project, what state it is in, etc. I don't mind creating a new document for this but I don't think this information should live in a how-to guide.

On the subject of simply pointing to another source of documentation, I would disagree. This repo here contains Magma's official documentation and through this new how-to guide, an operator would be deploying Magma. Even though operators are handled in another project, the software deployed is the same: The OCI images and go code that we are deploying are identical.

Reply:
And for reference, here are the principles I followed to base my views on how-to guides: https://diataxis.fr/how-to-guides/

Reply:
@maxhbr Can you please review my comments and either resolve the thread or provide direction?

Reply:
still think that it should highlight that there are known limitations.

If I find that document as is right now I would expect it to be a fully working and complete deployment. But as I understood that is not yet the case.

Please correct me if I am wrong here.

maybe:
```diff 
- > Charmed-Magma is in Beta and is not yet production ready.
+ > Charmed-Magma is in Beta and is not yet production ready or feature complete.
```
that should be a sufficient pointer.

Reply:
Understood, I just applied your recommendation.